### PR TITLE
Embed performance profile docs

### PR DESCRIPTION
**Pull request by yanirq**
Additions in this PR:
- performance addons opeator docs into NTO
- helper methods mentioned in docs to Makefile
- docs generator in Makefile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ any changes to node-level settings before applying the new profile. The
 containerized TuneD daemon handles termination signals by rolling back any
 node-level settings it has applied before gracefully shutting down.
 
+## Perfomance Profile Controller
+[Performance Addon controller](docs/performance_addon.md), previously known as Performance addons operator and now a part of the Node Tuning Operator, optimizes OpenShift clusters for applications sensitive to cpu and network latency.
```
Comment (jmencak) [inline README.md:25]: Can we keep formatting at least reasonably consistent in this `README.md`?  I try (an sometimes fail) to wrap (old school) at 80 characters.  The line presented here is way too long.

**Review by jmencak — COMMENTED** [review]
I wonder if we should keep the PAO docs under `docs/performanceprofile` or similar.  The NTO `README.md` file is getting big and we should probably spit it into `docs/` in some fashion.  Having a deeper directory structure would make this easier in my view.

**Comment by fromanirh** [comment]
LGTM

**Comment by jmencak** [comment]
Thank you for the changes, Yanir!
/lgtm
/approve

**Pull request merged**